libs/design-system/src/lib/Pages/SporkFAQPage/index.tsx

```diff
@@ -0,0 +1,62 @@
+  },
+  {
+    question: 'This is another question',
+    answer:
+      'Lorem ipsum dolor sit amet. Et officia Quis sed vero corrupti hic fuga asperiores? Qui provident dolor hic pariatur deserunt cum mollitia rerum et tempore sint non fugiat dolor a molestiae deserunt.\n\nAut cumque internos et accusantium Quis aut obcaecati minus vel esse dolores? Sed molestiae unde a nulla amet et debitis laborum aut quidem tempora. Id fugit quia id quia distinctio in minima internos quo laboriosam possimus.\n\nCum optio autem ut velit rerum sed culpa omnis ut deleniti deleniti. Et amet quos labore quia ut ullam dolor et consequatur sint! Ut minus nostrum ea eius voluptates quo minima itaque At repellendus saepe et quasi vitae sit quia illo eos cumque omnis.',
+  },
+  {
+    question: 'This is another question',
+    answer:
+      'Lorem ipsum dolor sit amet. Et officia Quis sed vero corrupti hic fuga asperiores? Qui provident dolor hic pariatur deserunt cum mollitia rerum et tempore sint non fugiat dolor a molestiae deserunt.\n\nAut cumque internos et accusantium Quis aut obcaecati minus vel esse dolores? Sed molestiae unde a nulla amet et debitis laborum aut quidem tempora. Id fugit quia id quia distinctio in minima internos quo laboriosam possimus.\n\nCum optio autem ut velit rerum sed culpa omnis ut deleniti deleniti. Et amet quos labore quia ut ullam dolor et consequatur sint! Ut minus nostrum ea eius voluptates quo minima itaque At repellendus saepe et quasi vitae sit quia illo eos cumque omnis.',
+  },
+];
+
+export function SporkFAQPage() {
+  return (
+    <main className="bg-zinc-50 px-4 pb-48 dark:bg-black md:bg-white md:px-0 md:pt-16">
+      <div className="grid grid-cols-1 items-start pb-6 md:flex md:pb-12">
+        <div className="order-2 grow">
+          <h1 className="pb-14 font-display text-[44px] font-bold leading-snug md:pb-6 md:font-sans md:text-4xl">
+            <span className="hidden md:block">Spork FAQ</span>
+            <span className="block md:hidden">FAQ</span>
+          </h1>
+          <Attribution
+            updatedDate="23/3/2022"
+            authorIcon="https://avatars.githubusercontent.com/u/62387156?s=64&v=4"
+            authorName="@maxxP"
+            otherAuthorsCount={12}
+            readMinutes={4}
+            difficulty="Beginners"
+          />
+        </div>
+        <div className="order-1 md:order-2">
+          <button className="mt-2 flex items-center rounded-lg border-primary-blue py-4 text-primary-blue dark:border-primary-purple dark:text-primary-purple md:m-0 md:border md:px-5">
```

Comment:
@neilZon I updated the Button component to use left/right icons, you can use it like this after rebasing from main:

```
      <Button variant="secondary" prev={true}>
          Back to Network
      </Button>
      ```